### ISSUE TITLE
tour: Why is 'hello' printed 5 times?

### ISSUE THREAD

**Issue by IvanMartinez** [golang/tour#60]
Context: http://127.0.0.1:3999/concurrency/1

Looking at the code I expected 'hello' to be printed one time and 'world' five times, but both are printed five times. What puzzled me more is that I swaped the lines:

```
    say("hello")
    go say("world")
```

And then 'hello' was printed five times and 'world' none. 
This step of the tour needs a better explanation of how Goroutines work, IMO.

**Comment by alebaffa** [comment]
'world' is not printed because the goroutines has no time to execute before the main() ends. 
You're right, this is not well explained here, but it's also true that swapping those lines is not something requested in this exercise. 
The goal of the Tour is to give a very simple and quick overview of the basics of the language, and in case people start to dig more in depth, the best way is to study the other material mentioned in the end of the Tour itself, or ask somewhere else. It's not in the scope of the Tour to answer every kind of questions.
Hope this clarify.
Please, also consider if this issue needs to stay open or could be closed. 
Thank you!

**Comment by PercyPham** [comment]
I got the same question, but then I realize that I misunderstood the example too.

@alebaffa explained that, but I still want to explain it again for anyone got confused like I did:

For short: remove the word `go` in the original problem:
```
say("world")
say("hello")
```
You'll see that the word "world" is printed 5 times, and then the word "hello" is printed 5 times
```
world
world
world
world
world
hello
hello
hello
hello
hello
```

And when you put back the word `go` before `say("world")`. Go will make another "goroutine" to run that  `say("world")`. Therefore, we have two goroutines running in our program now: 
- `main()` 
- `say("world")`

The result will be like:

```
world
hello
world
hello
world
hello
world
hello
world
hello
```

And when @IvanMartinez swapped two lines of code in `main()`
```
func main() {
	say("hello")
	go say("world")
}
```
Then the `main()` will run `say("hello")` first, printed 5 lines of "hello".
```
hello
hello
```
And then `main()` will execute the line `go say("world")`, it creates another goroutine to run it
Then main immediately try to run the next line in `main()`. However
